src/command/AllCommands.cxx

```diff
@@ -185,7 +185,8 @@ static constexpr struct command commands[] = {
 	{ "stats", PERMISSION_READ, 0, 0, handle_stats },
 	{ "status", PERMISSION_READ, 0, 0, handle_status },
 #ifdef ENABLE_SQLITE
-	{ "sticker", PERMISSION_ADMIN, 3, -1, handle_sticker },
+	{ "sticker", PERMISSION_ADMIN, 1, -1, handle_sticker },
```

Comment:
What happened to this line?

Reply:
Sorry, this was relict of a previous attempt to implement this. Is now fixed.